lib/routes/reviews.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable new-cap */
+const router = require('express').Router();
+const Review = require('../models/review');
+
+router
+  .post('/', (req, res, next) => {
+    Review.create(req.body)
+      .then(review => res.json(review))
+      .catch(next);
+  })
+
+  .get('/:id', (req, res, next) => {
+    Review.findById(req.params.id)
```

Comment:
you don't need a find review by id route - looks like you moved the functionality that should be in the reviewer's get by id route to the review routes (and switched it around to pull the reviewer into review rather than the reviews into the reviewer).